lib/pubnub/validators/common_validator.rb

```diff
@@ -69,6 +71,23 @@ def validate_uuid(uuid)
           )
         end
       end
+      deprecate :validate_uuid, :validate_user_id, 2023, 1
```

Comment:
@kleewho there is no need to duplicate whole method twice. From `validate_uuid(uuid)` call `validate_user_id(user_id)` and it will be enough.

Reply:
It's used in `events.rb` and it validates the `:uuid` parameter that users can override. The error message will be different misleading the users. Imagine that someone would like to override `:uuid` but get's `user_id invalid` information. What `user_id`?

Reply:
I removed it completely. I think it was actually used in just one place